server/gitrest/Dockerfile

```diff
@@ -2,7 +2,7 @@
 # Licensed under the MIT License.
 # DisableDockerDetector "No feasible secure solution for OSS repos yet"
 
-FROM node:18.20.5-bookworm-slim@sha256:c4a278056a0f79abf472d912bbf5af6c874126ae450faad9df52069e9ad78335 AS runnerbase
+FROM node:20.19.0-bookworm-slim@sha256:d6e4ec9eaf2390129b5d23904d07ae03ef744818386bcab3fc45fe63405b5eb2 AS runnerbase
```

Comment:
Looks like this images has some active CVEs: https://hub.docker.com/layers/library/node/20.19.0-bookworm-slim/images/sha256-b0938698f04da129ca7f58f83ac1bac982eaec0d29ff799ce92ae520b0bda566

Looks like at least the high severity ones at the same as the old image though ( https://hub.docker.com/layers/library/node/18.20.5-bookworm-slim/images/sha256-86a165f3f6df9dede787588974ec5c6e946649607e06a5ba8eedd12dcf3c241e?context=explore ) so I guess it's not a regression. Still something that should prabably be looked into though.

Reply:
Switched the sha to 990084211ea3d72132f286a038bf27e0f79b8cad532ae623b090f3486490e62e which reduces the number of vulns to 1